api/controllers/todoListController.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+const Todo = mongoose.model('Todos')
+
+exports.list_all_todos = (req, res) => {
+  Todo.find({}, (err, todos) => {
+    if(err) {
+      res.send(err);
+    }
+    res.json(todos);
+  });
+};
+
+exports.create_todo = (req, res) => {
+  const new_todo = new Todo(req.body);
+  new_todo.save((err, todo) => {
```

Comment:
don't think there's a need for the second callback parameter here (see code above).